nix/status-go/mobile/default.nix

```
@@ -1,19 +1,30 @@
 { callPackage, meta, source, goBuildLdFlags }:
 
-{
-  android = callPackage ./build.nix {
+let
+  android = {abis ? [ "armeabi-v7a" "arm64-v8a" "x86" ]}: callPackage ./build.nix {
     platform = "android";
     platformVersion = "23";
-    targets = [ "android/arm" "android/arm64" "android/386" ];
     outputFileName = "status-go-${source.shortRev}.aar";
+    # Hide different arch naming in gomobile from Android builds.
+    targets = let
+      abiMap = {
+        "armeabi-v7a" = "android/arm";
+        "arm64-v8a" = "android/arm64";
+        "x86" = "android/386";
+        "x86_64" = "android/386";
+        };
+      in map (arch: abiMap."${arch}") abis;
```

Hm, what do you think about:
```suggestion
    targets = let
      abiMap = {
        "armeabi-v7a" = "arm";
        "arm64-v8a"   = "arm64";
        "x86"         = "386";
        "x86_64"      = "amd64";
      };
      in map (arch: "android/" + abiMap."${arch}") abis;
```
Notice I also fixed wrong value for `x86_64`.